gamemodes/terrortown/gamemode/shared/lang/german.lua

```diff
@@ -1164,6 +1165,9 @@ L.f1_bind_description = "Klicke und drücke einen Knopf um ihn zu setzen."
 L.f1_bind_reset_default_description = "Auf Standardwert zurücksetzen."
 L.f1_bind_disable_description = "Deaktiviere diese Bindung."
 
+L.f1_bind_weaponswitch = "Waffe Wechseln"
+L.f1_bind_sprint = "Sprinten"
```

Comment:
See above

Reply:
"There are some lines that are not a part of this PR theoretically. As small changes, they are okay. But think about the last PR, this should be avoided." (first comment)